app/shop.py

```diff
@@ -0,0 +1,27 @@
+import datetime
```

Comment:
The import statement is correct according to the checklist requirement: 'Make sure you use exactly `import datetime`, not the other way of import.'